test/evp_extra_test.c

```diff
@@ -3982,6 +4019,77 @@ static int test_evp_init_seq(int idx)
     return testresult;
 }
 
+/*
+ * Test re-initialization of cipher context without changing key or iv.
+ * The result of both iteration should be the same.
+ */
+static int test_evp_reinit_seq(int idx)
+{
+    int outlen1, outlen2, outlen_final;
+    int testresult = 0;
+    unsigned char outbuf1[1024];
+    unsigned char outbuf2[1024];
+    const EVP_INIT_TEST_st *t = &evp_reinit_tests[idx];
+    EVP_CIPHER_CTX *ctx = NULL;
+    EVP_CIPHER *type = NULL;
+    char *errmsg = NULL;
+
+    ctx = EVP_CIPHER_CTX_new();
```

Comment:
Changes look fine.. Now that the tests work maybe simplify them. The err_msg lines are not required since the TEST_ macros tell you where it failed.

Just change the first line to if (!TEST_ptr(ctx =...))

Reply:
Done.